code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -255,3 +255,21 @@
 		toggle_folded(user)
 		return FALSE
 	return ..()
+
+/obj/item/gun/projectile/shotgun/foldable/cameragun
+	name = "camera"
+	desc = "A polaroid camera"
+	desc_extended = "Upon closer inspection, this is not a camera at all, but a 9mm firearm concealed inside the shell of one, which can be deployed by pressing a button."
```

Comment:
This should probably be hidden and only shown if the camera is inspected in hand or the likes, this way anyone from afar can shift click you, select the camera if you have it in hand, then check the extended description and know it's a firearm

Reply:
If there's a way to do that I'd be happy to make that happen

Reply:
> If there's a way to do that I'd be happy to make that happen

/atom/proc/examine_fluff(mob/user, distance, is_adjacent, infix = "", suffix = "")

You can override this proc to override what extended description is displayed